sapling-dev-server/Dockerfile

```diff
@@ -21,6 +21,10 @@ RUN npm config set unsafe-perm true
 
 COPY . .
 
+RUN cd /saplings/circuits && \
+  npm install && \
+  yarn deploy
```

Comment:
I believe it is this way in the Grid saplings, but to be consistent you probably want to do `npm run deploy`. We had to switch to npm due to git dependencies not working properly with yarn.

Reply:
Updated